sport/app/cricket/feed/throttledTask.scala

```diff
@@ -1,42 +1,43 @@
 package cricket.feed
```

Comment:
I've rewritten the throttled task since Throttled actor is deprecated. I will cherry-pick this into 2.11 and release separately